ddp-workspace/projects/ddp-sdk/src/lib/services/sessionMemento.service.ts

```diff
@@ -110,6 +116,19 @@ export class SessionMementoService implements OnDestroy {
         this.sessionSubject.next(session);
     }
 
+    public updateSessionLocale(languageCode: string): void {
+      if (this.session === null) {
+        return;
+      }
+
+      const updatedSession = Object.assign({}, this.session, { locale: languageCode });
+
+      localStorage.setItem(this.SESSION_KEY, JSON.stringify(updatedSession));
+      localStorage.setItem(this.TOKEN_KEY, updatedSession.idToken);
+
+      this.sessionSubject.next(updatedSession);
```

Comment:
You can update session using existing method:
```suggestion
      this.updateSession(updatedSession);
```

Reply:
Yep, removed `updateSessionLocale` and switched to `updateSession`.